proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java

```diff
@@ -335,7 +340,7 @@ public void handle(PluginMessage pluginMessage) throws Exception
             throw CancelSendSignal.INSTANCE;
         }
 
-        if ( pluginMessage.getTag().equals( PluginMessage.BUNGEE_CHANNEL_LEGACY ) )
```

Comment:
Unneeded because https://github.com/SpigotMC/BungeeCord/blob/e3c7fd8cc59e89e02abecab1d60841dd127c79ef/protocol/src/main/java/net/md_5/bungee/protocol/packet/PluginMessage.java#L27